package.json

```diff
@@ -101,6 +101,7 @@
     "http-proxy": "^1.16.2",
     "istanbul": "0.4.5",
     "mocha": "^5.1.1",
+    "rewire": "3.0.2",
```

Comment:
❓This appears to be unused?

Reply:
I deleted it because it became unnecessary.